inbox/types.go

```diff
@@ -123,6 +123,24 @@ func (i ObjectTypeIdentifier) EnumIndex() int {
 	return int(i)
 }
 
+const (
+	GroupAddStatusFail    = "failed"
+	GroupAddStatusSuccess = "success"
+)
+
+type GroupMembersInfo struct {
+	Address string `json:"address"`
+	Status  string `json:"status"`
+	Reason  string `json:"reason"`
+}
+
+type GroupsWithMembersInfo struct {
+	Address     string             `json:"address"`
+	Status      string             `json:"status"`
+	Reason      string             `json:"reason"`
+	MembersInfo []GroupMembersInfo `json:"membersInfo,omitempty"`
+}
+
```

Comment:
why not moving these structs in apis file instead.

Reply:
Done